src/components/Charts/parseResponse.js

```diff
@@ -132,12 +130,30 @@ const parseResponse = (state, province, response) => {
         total: sumBy(filterResponse, p => p.dwellings)
     };
 
-    const typesName = uniq(flatten(
-        filterResponse.map(p => p.dwellingTypes.map(type => type.type || DWELLINGS_WITHOUT_TYPE))
-    ));
+    const dwellingTypesNames = [];
+    const dwellingTypesValues = [];
+    const responseDwellingTypes = filterResponse.map(p => p.dwellingTypes);
```

Comment:
Viene de arriba así igual, pero por qué no usar `map` de lodash?

Reply:
Se arreglo en https://github.com/indec-it/react-address-commons/pull/33